feat(table): enables sorting (#904)
uber-web/baseui: documentation-site/pages/components/table.mdx

  <Basic />
</Example>

<Example
  title="Table with vertical scroll"
  path="examples/table/vertical-scroll.js"
>
  <VerticalScroll />
</Example>

<Example
  title="Table with horizontal scroll"
  path="examples/table/horizontal-scroll.js"
>
  <HorizontalScroll />
</Example>

> <Example title="Table with horizontal scroll" path="examples/table/sortable.js">

Author: ```suggestion
<Example title="Table with sorting by a column" path="examples/table/sortable.js">
```
